go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/annakrystalli/rse-workshop-materials-template
 go 1.14
 
 require (
-	github.com/dzello/reveal-hugo v0.0.0-20210413081415-39511a646b9c // indirect
-	github.com/matcornic/hugo-theme-learn v0.0.0-20210331234833-d198cbe65f06 // indirect
+	github.com/dzello/reveal-hugo v0.0.0-20220710134748-1c7822db0cf2 // indirect
```

Comment:
Are these just sub-minor version changes?

Reply:
It seems that way doesn't it, hugo modules are still a bit confusing to me.